### DISCUSSION TITLE
Sonoff Multiswitch does not work

### DISCUSSION THREAD

**Discussion by 74mzsolt** [matt8707/ha-fusion#461] [Q&A]
hello,

I have 2-3 pieces of Sonoff 4 channel wireless switches, the pro and the non-pro as well. As I recognize, the normal 1 channel sonoff switch works fine, but this 4 channel wifi switch does not work at all (the toggle function does not pull)

If I put my switch action to "switch.turn_off" or "switch.turn_on" it also works only for 1 channel sonoff device.

I really like the design and the very easy functionality but unfortunately (for me) without sonoff support I cannot use it.

**Comment by matt8707** [comment]
Hi!

Can you toggle the switch with `switch.toggle` in HA?

[![Open your Home Assistant instance and show your service developer tools with a specific service selected.](https://my.home-assistant.io/badges/developer_call_service.svg)](https://my.home-assistant.io/redirect/developer_call_service/?service=switch.toggle)

Do you have any special integration to make it work in your HA dashboard? Is it something special like mqtt?

**Comment by 74mzsolt** [comment]
Hello
Yes, there it works fine

Mattias Persson ***@***.***> ezt írta (időpont: 2024. máj.
14., K 19:48):

**Comment by 74mzsolt** [comment]
Its not mqqt but a sonoff local integration

Thank you for your nice work and Effort

Mattias Persson ***@***.***> ezt írta (időpont: 2024. máj.
14., K 19:48):

**Comment by matt8707** [comment]
How about `homeassistant.toggle`, this is what is used currently. Any browser console errors?

[![Open your Home Assistant instance and show your service developer tools with a specific service selected.](https://my.home-assistant.io/badges/developer_call_service.svg)](https://my.home-assistant.io/redirect/developer_call_service/?service=homeassistant.toggle)

**Comment by 74mzsolt** [comment]
yes from developer mode it works fine


Mattias Persson ***@***.***> ezt írta (időpont: 2024. máj.
14., K 19:48):

**Comment by matt8707** [comment]
If it works in dev tools it should work in fusion, because it's the same service call 🤔
https://github.com/matt8707/ha-fusion/blob/c27033b623bc7e28ab31d255ccfba6716465a543/src/lib/Modal/SwitchModal.svelte#L19

**Comment by 74mzsolt** [comment]
uninstalled :)
